ci/deps/actions-38-minimum_versions.yaml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - pytest-xdist>=1.31
   - psutil
   - pytest-asyncio>=0.17
-  - boto3
```

Comment:
`boto3` is just a testing dependency so I don't think it should have a min version. I think you can just remove this from `setup.cfg` and leave this here

Reply:
Sure thing will do.